agent/functional_tests/util/utils.go

```diff
@@ -115,7 +115,7 @@ type AgentOptions struct {
 }
 
 // PLatform Independent piece of Agent Startup. Gets executed on both linux and Windows.
```

Comment:
While you're here, can you also fix the comment? `// verifyIntrospectionAPI verifies ...`